src/assets/scripts/client/aircraft/StripView/StripViewModel.js

```diff
@@ -483,6 +532,9 @@ export default class StripViewModel extends BaseModel {
         this.$arrivalAirportView.text(this._arrivalAirport);
         this.$alternateAirportView.text(this._alternateAirport);
         this.$flightPlanView.text(this._flightPlan);
+        this.$remarks.text(this._remarks);
+        this.$runway.text(this._runwayInformation.name);
+        this.$runway.toggleClass(STRIP_VIEW_PREPLANNING_CLASS, !this._runwayInformation.assigned);
```

Comment:
I'm a little wary about this one.  My goal for this method was to simple accept data and render it.  Modifying css and such feels like it should happen somewhere else within this class.

Reply:
Seems like the only place that it could work. Maybe he means to put it in _layout()? That's another place where a style class is applied, and the docblock says it is run every redraw-- but it actually isn't. So without having redraw() call layout() each time, the only place it really could go is in redraw()...

Reply:
sorry guys, phone PRs are not the best PRs ☹️ 

The `.toggleClass` mentioned here should live in `_.updateStripView()`

Reply:
done